dashboard/src/pages/TreeDetails/TreeDetailsFilter.tsx

```diff
@@ -141,47 +164,55 @@ const CheckboxSection = ({
         subtitle: intl.formatMessage({ id: 'filter.branchSubtitle' }),
         items: parsedFilter.branches,
         onClickItem: (value: string): void => {
-          setDiffFilter(old => changeFilterValue(old, 'branches', value));
+          setDiffFilter(old =>
+            changeCheckboxFilterValue(old, 'branches', value),
+          );
         },
       },
       {
         title: intl.formatMessage({ id: 'filter.buildStatus' }),
         subtitle: intl.formatMessage({ id: 'filter.statusSubtitle' }),
         items: parsedFilter.status,
         onClickItem: (value: string): void => {
-          setDiffFilter(old => changeFilterValue(old, 'status', value));
+          setDiffFilter(old => changeCheckboxFilterValue(old, 'status', value));
         },
       },
       {
         title: intl.formatMessage({ id: 'filter.bootStatus' }),
         subtitle: intl.formatMessage({ id: 'filter.statusSubtitle' }),
         items: parsedFilter.bootStatus,
         onClickItem: (value: string): void => {
-          setDiffFilter(old => changeFilterValue(old, 'bootStatus', value));
+          setDiffFilter(old =>
+            changeCheckboxFilterValue(old, 'bootStatus', value),
```

Comment:
is there any way we can group the possibilities together, like an enum, instead of using it hard-coded? `bootStatus, testStatus, configs, arch, etc`

Reply:
These are name of object properties, so i don't think we will have benefits from that grouping